store/adapters/rdbms/upgrade_tables.go

```diff
@@ -696,3 +697,22 @@ func tableResourceActivityLog() *Table {
 		AddIndex("ts", IColumn("ts")),
 	)
 }
+
+func tableDataPrivacyRequests() *Table {
+	return TableDef("data_privacy_requests",
+		ID,
+		ColumnDef("name", ColumnTypeIdentifier),
+		ColumnDef("request_type", ColumnTypeText, ColumnTypeLength(handleLength)),
+		ColumnDef("status", ColumnTypeInteger),
+		ColumnDef("requested_at", ColumnTypeTimestamp),
+		ColumnDef("requested_by", ColumnTypeTimestamp),
+		ColumnDef("completed_at", ColumnTypeTimestamp),
+		ColumnDef("completed_by", ColumnTypeTimestamp),
+
+		AddIndex("request_type", IColumn("request_type")),
```

Comment:
index overload? do we really need all of them?

Reply:
No